### PR TITLE
Abhiz

### PR DESCRIPTION
**Pull request by AbhinandSuresh**
Please review my code

### CODE REVIEW

**Comment by AbhinandSuresh** [comment]
please review AWMS Monitor code

File: AWMScontroller.ino [hunk 1]
```diff
@@ -0,0 +1,158 @@
+#include <SPI.h>              // include libraries
+#include <LoRa.h>
+
+const long frequency = 433E6;  // LoRa Frequency
```
Comment (greatinbaby) [inline AWMScontroller.ino:4]: Better to use #define for contants

File: AWMScontroller.ino [hunk 2]
```diff
@@ -0,0 +1,158 @@
+#include <SPI.h>              // include libraries
+#include <LoRa.h>
+
+const long frequency = 433E6;  // LoRa Frequency
+
+const int csPin = 15;          // LoRa radio chip select
+const int resetPin = 16;        // LoRa radio reset
```
Comment (greatinbaby) [inline AWMScontroller.ino:7]: same here

File: AWMScontroller.ino [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+void LoRa_rxMode() {
+  LoRa.disableInvertIQ();               // normal mode
+  LoRa.receive();                       // set receive mode
+}
+
+void LoRa_txMode() {
+  LoRa.idle();                          // set standby mode
+  LoRa.enableInvertIQ();                // active invert I and Q signalsLoRa.idle(); 
+  //delay(5);
```
Comment (greatinbaby) [inline AWMScontroller.ino:52]: remove unwanted comments

File: AWMScontroller.ino [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+  if(isTxdone)
+  {
+   LoRa_rxMode();
+    isTxdone = false;
+  }
+   
+  if(!isPktAvailable)
+    return;
+    
+
```
Comment (greatinbaby) [inline AWMScontroller.ino:66]: remove unwanted space

File: AWMScontroller.ino [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+   LoRa_rxMode();
+    isTxdone = false;
+  }
+   
+  if(!isPktAvailable)
+    return;
+    
+ 
+  isPktAvailable = false;
+  delay(1000);
```
Comment (greatinbaby) [inline AWMScontroller.ino:68]: why delay of 1000?

File: AWMScontroller.ino [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+  LoRa_sendMessage("HELLO");
```
Comment (greatinbaby) [inline AWMScontroller.ino:69]: what is this message

File: AWMScontroller.ino [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+  if (loraval > 130)
```
Comment (greatinbaby) [inline AWMScontroller.ino:70]: use #defined constants

Comment (AbhinandSuresh) [reply AWMScontroller.ino:70]: ok.

File: AWMScontroller.ino (hunk 6 above)
Comment (AbhinandSuresh) [reply AWMScontroller.ino:69]: code under test.

File: AWMScontroller.ino (hunk 4 above)
Comment (AbhinandSuresh) [reply AWMScontroller.ino:66]: ok

File: AWMScontroller.ino (hunk 1 above)
Comment (AbhinandSuresh) [reply AWMScontroller.ino:4]: noted

File: AWMScontroller.ino (hunk 6 above)
Comment (greatinbaby) [reply AWMScontroller.ino:69]: please be careful about such mistakes going forward

**Pull request closed without merging**